sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -206,6 +212,48 @@ def test_partial_dependence_helpers(est, method, target_feature):
     assert np.allclose(pdp, mean_predictions, rtol=rtol)
 
 
+@skip_if_32bit
```

Comment:
Do you have any idea why is this necessary? The differences seems very large. Maybe you should try with a non-random dataset (e.g. make_regression) so that the tree and the GBRT model have some more numerically stable structure?

Reply:
I suspect it comes from a split with equal gain, but I'm not sure. I was really confused. Now I'm properly testing the recursion method so maybe we can remove this